demo/x-select.html

```diff
@@ -21,15 +21,15 @@
       }
     </style>
     <div on-tap="open">
-      <content select=".dropdown-trigger"></content>
+      <slot name="dropdown-trigger"></slot>
     </div>
     <iron-dropdown id="dropdown"
       vertical-align="[[verticalAlign]]"
       horizontal-align="[[horizontalAlign]]"
       disabled="[[disabled]]"
       open-animation-config="[[openAnimationConfig]]"
       close-animation-config="[[closeAnimationConfig]]">
-      <content select=".dropdown-content"></content>
+      <slot></slot>
```

Comment:
(Discussion in paper-menu-button for reference: https://github.com/PolymerElements/paper-menu-button/pull/105/files#r109993878)
I think we should convert this to a named slot like `dropdown-trigger`.